parktain/main.py

```diff
@@ -65,12 +65,23 @@ def checkins_reminder():
     date = datetime.now().strftime('%d %B, %Y')
     bot.speak('Morning! What are you doing on {}!'.format(date), "#checkins")
 
-@bot.listen_for(lambda user, channel, message: True)
+@bot.listen_for(lambda user, channel, message: True, target_channel='clickbaits')
 def logger(user, channel, message):
     message_log = Message(user_id=user, channel_id=channel, message=message, timestamp=datetime.now())
     session.add(message_log)
     session.commit()
 
+    # Check for presence of HyperLink in message
+    if user == bot.id:
+        return
+    for word in message.split():
+        try:
+            o = urlparse(word[1:-1])
```

Comment:
From slack, when there is a url in message I get url enclosed between <url>. So have to do this to skip them.
Example: '<http://qz.com/620076/beijing-is-banning-all-foreign-media-from-publishing-online-in-china/>'